src/libraries/Microsoft.Extensions.Logging.Debug/src/DebugLogger.cs

```diff
@@ -31,8 +31,7 @@ public DebugLogger(string name)
         /// <inheritdoc />
         public bool IsEnabled(LogLevel logLevel)
         {
-            // If the filter is null, everything is enabled
-            // unless the debugger is not attached
+            // Everything is enabled unless the debugger is not attached
```

Comment:
>Everything [](http://example.com/codeflow?start=15&length=10)

nit: I agree the old comment is unclear but it was providing extra info that LogLevel.Debug is the highest level before `None` and that what refer to filter in the old comment.

Reply:
I don't think so.  This type used to have a member named filter.  That was removed but the comment was not updated: https://github.com/dotnet/extensions/blob/ab493da045ef1e645ff52675adb177ba7a503ce0/src/Logging/Logging.Debug/src/DebugLogger.cs#L46-L50